test/functional/legacy/delete_spec.lua

```diff
@@ -114,4 +115,16 @@ describe('Test for delete()', function()
     eq(0, eval("delete('Xdir4/Xfile')"))
     eq(0, eval("delete('Xdir4', 'd')"))
   end)
+    it('delete errors', function()
+        local err = exc_exec("call delete('''')")
+        -- Expected
+        -- neq(err:find('E474:'), nil)
+        -- Actual
+        eq(0, err)
```

Comment:
@janlazo do you know offhand what the problem here might be? Maybe `exc_exec` is not a correct function to use?